<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DbViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icons/main.png</normaloff>images/icons/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableViewMain"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Обзор</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Выбор данных</string>
     </property>
     <addaction name="actionOpenTBooks"/>
     <addaction name="actionOpenTLibraries"/>
     <addaction name="action"/>
     <addaction name="action_2"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="actionAddRow"/>
    <addaction name="actionUpdateRow"/>
    <addaction name="actionDeleteRow"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelpBrowser"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string>Status bar</string>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAddRow">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="actionDeleteRow">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionUpdateRow">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="actionOpenTBooks">
   <property name="text">
    <string>В ремонте</string>
   </property>
  </action>
  <action name="actionHelpBrowser">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="actionOpenTLibraries">
   <property name="text">
    <string>Заказы</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Кадры</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Графики</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Склад</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Товары на складе</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Поставки</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Поставщики</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
